src/main/java/racingcar/RacingCars.java

```diff
@@ -0,0 +1,51 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+class RacingCars {
+    private static final String MOVEMENT_SITUATION = "-";
+
+    private ArrayList<Car> cars;
+
+    private RacingCars(ArrayList<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static RacingCars from(ArrayList<Car> cars) {
+        return new RacingCars(cars);
+    }
+
+    public int size() {
+        return this.cars.size();
+    }
+
+    public void goForward(final int index, final int randomNumber) {
+        if(isNumberMoreThanFour(randomNumber)) {
+            this.cars.set(index, this.cars.get(index).changePosition(MOVEMENT_SITUATION));
+        }
+    }
+
+    public String getNameByIndex(final int index){
```

Comment:
```suggestion
    public String getNameByIndex(final int index) {
```

매의 눈 @_@ 


IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.